src/sb_test.h

```diff
@@ -79,6 +79,7 @@ typedef struct sb_test_buf_t {
 static const sb_test_buf_t sb_test_buf_init = { .buf = NULL, .len = 0 };
 
 extern _Bool sb_test_open(const char* name, FILE** handle);
+extern _Bool sb_test_read_line(FILE* const handle, sb_test_buf_t* const line);
```

Comment:
Remove these `const`s - they don't add any value in a prototype.

Reply:
This one still needs to be resolved.

Reply:
I didn't mention it in the commit message but I also fixed this in the most recent commit.